jans-cedarling/bindings/cedarling_python/example_files/policy-store.json

```diff
@@ -2,15 +2,15 @@
     "cedar_version": "v2.4.7",
     "cedar_policies": {
         "840da5d85403f35ea76519ed1a18a33989f855bf1cf8": {
-            "description": "simple policy example for principal workload",
+            "description": "simple policy example for pricipal workload",
             "creation_date": "2024-09-20T17:22:39.996050",
             "policy_content": "cGVybWl0KAogICAgcHJpbmNpcGFsIGlzIEphbnM6Oldvcmtsb2FkLAogICAgYWN0aW9uIGluIFtKYW5zOjpBY3Rpb246OiJVcGRhdGUiXSwKICAgIHJlc291cmNlIGlzIEphbnM6Oklzc3VlCil3aGVuewogICAgcHJpbmNpcGFsLm9yZ19pZCA9PSByZXNvdXJjZS5vcmdfaWQKfTs="
         },
         "444da5d85403f35ea76519ed1a18a33989f855bf1cf8": {
-            "description": "simple policy example for principal user",
+            "description": "simple policy example for pricipal user",
```

Comment:
```suggestion
            "description": "simple policy example for principal user",
```

Reply:
Fixed in [commit](https://github.com/JanssenProject/jans/pull/10016/commits/ca2e503d76279f88ef704b814b4e4d16ab621697)